app/Http/Controllers/BookController.php

```diff
@@ -262,6 +262,16 @@ public function destroy($id)
     {
         try {
             $book = Book::findOrFail($id);
```

Comment:
đoạn này em đang bị N+1 rất nhiều.
em dùng with() ở mấy chỗ find luôn đi
```suggestion
            $book = Book::with(reviews, review.comments, review.comment.like)findOrFail($id);
```
cú pháp chị viết chưa đúng, em sửa lại nhé